src/components/AnnotationPopover/AnnotatorLabel.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../variables';
+
+.ba {
+    .ba-annotation-label {
+        line-height: 23px;
+        padding: 3px 0;
+    }
+
+    .ba-annotation-label ~ .ba-action-controls {
+        border-left: 1px solid $see-see;
+        margin-left: 10px;
+    }
+
+    // MOBILE & TABLET CSS
+    @media #{$mobile}, #{$tablet} {
+        .ba-annotation-label {
+            align-self: flex-start;
```

Comment:
same question as above regarding achieving it with align-items or justify-content intead